docs/operator-manual/user-management/identity-center.md

```diff
@@ -0,0 +1,81 @@
+# Identity Center (AWS SSO)
+
+!!! note "Are you using this? Please contribute!"
+    If you're using this IdP please consider [contributing](../../developer-guide/site.md) to this document.
+
+A working Single Sign-On configuration using Identity Center (AWS SSO) has been achieved using the following method:
+
+* [SAML (with Dex)](#saml-with-dex)
+
+## SAML (with Dex)
+
+1. Create a new SAML application in Identity Center and download the certificate.
+    * ![Identity Center SAML App 1](../../assets/identity-center-1.png)
+    * ![Identity Center SAML App 2](../../assets/identity-center-2.png)
+2. Click `Assign Users` after creating the application in Identity Center, and select the users or user groups you wish to grant access to this application.
+    * ![Identity Center SAML App 3](../../assets/identity-center-3.png)
+3. Copy the Argo CD URL into the `data.url` field in the `argocd-cm` ConfigMap.
+
+<!-- markdownlint-disable MD046 -->
+```yaml
+data:
+  url: https://argocd.example.com
+```
+
+4. Configure Attribute mappings.
+
+    !!! note
+       Group attribute mapping is not officially supported in the AWS docs, however the workaround is currently working.
```

Comment:
Sorry about how the docs are rendered.   Please see lines #2-5 and note the spacing and indentations.
```yaml
4. Configure Attribute mappings.

!!! note
    Group attribute mapping is not officially supported in the AWS docs, however the workaround is currently working.

  * ![Identity Center SAML App 4](../../assets/identity-center-4.png)
  * ![Identity Center SAML App 5](../../assets/identity-center-5.png)
 ```

Reply:
I am trying to make the change that you suggested but it doesn't look good. I am attaching an image to see the differences.

In red is the change you suggested and in yellow is the actual code. The rest of the document looks the same.
![IMG_20231026_211947](https://github.com/argoproj/argo-cd/assets/33123154/a5b5a107-da13-44cb-9dff-8dee01a30ff2)

Reply:
You can run locally `make build-docs` & `make serve-docs` then visit localhost:3000 to see the changes you have made.   It is also possible to see the changes you made to the docs after a commit by clicking the Details section of `docs/readthedocs.org:argo-cd` in the CI checks.
![Screenshot 2023-10-26 at 1 22 17 PM](https://github.com/argoproj/argo-cd/assets/76892343/fac68f85-d7ca-41a3-be89-84866d7e92a1)

After making the changes to the `!!! note` section, it should be rendered similar to this:

![Screenshot 2023-10-26 at 1 29 24 PM](https://github.com/argoproj/argo-cd/assets/76892343/9b75615c-6bf7-4db2-9815-e19fa17f7d98)


Thank you so much for your patience.  If its to frustrating I will go ahead an accept the PR as is, then submit a fix for it.  Let me know if you would like to continue to fix this.

Reply:
@34fathombelow  Thank you very much for your help and your patience.

I have already understood you, I think that the format of the note is ok, however, I can't get the order of the numerical list right.

Reply:
No problem!  It looks like this code block is causing it:

```yaml
data:
  url: https://argocd.example.com
```
Most of our documentation uses headers vs numbered list.  Nothing wrong with the standard way you did it, but  I was able to come up with a workaround by indenting the code block by 4 tabs.   You have done enough work, we can fix this later.  If you want to make the changes feel free, if not I will merge as is later today.

![Screenshot 2023-10-27 at 4 31 03 AM](https://github.com/argoproj/argo-cd/assets/76892343/85f20bdd-dc9f-4313-a5fa-d3aa7e493b86)

Reply:
I have applied your suggestion, although it still doesn't follow the order, maybe the best thing to do is to merge it like this and then solve the visualization.